gradle/rat.gradle

```diff
@@ -63,6 +63,7 @@ rat {
 
     // working directories
     '**/locator/**',
+    '**/server/**',
```

Comment:
This was actually causing the existing `putall` example to fail if you ran it as a standalone example.  The `putall` example (and now the durableMessaging) example both use a single server named "server", so Rat needs to ignore any files generated for the "server" folder.